### ISSUE TITLE
Request for setting individual object detections with MQTT

### ISSUE THREAD

**Issue by MrAnonymous722** [blakeblackshear/frigate#2329]
**Describe what you are trying to accomplish and why in non technical terms**
I want to be able to set the state of detection for specific objects so that I can turn off detection, for example, for cars on my driveway camera when I have visitors over but still want to know if someone walks up to my house. 

**Describe the solution you'd like**
Much like how we can currently set the state of detect, record and clips with MQTT, it would be neat if the same was implemented for specific objects.

**Describe alternatives you've considered**
I have tried to make zones, but with the beta I end up with very long clips regardless for cars. I currently just turn off detection entirely on my driveway camera when I have visitors as to not have a bunch of/really long clips recorded throughout their stay.

**Additional context**
N/A

**Comment by sygys** [comment]
I think what i want is something similar. 

I am experimenting with zones and person and book detection in the. But i ran into the problem that even when setting

```
detect:
  max_disappeared: 400
```
this does not work for zones. so the motion detection goes on and off every few seconds within a zone. 

Also a great option would be that when an object isnt moving but still in the frame it keeps being detected. So when me and my girl sitting on the couch whatching a movie the camera will keep detecting 2 persons and not only when we move.

**Comment by blakeblackshear** [comment]
@sygys I don't think you understand what max_disappeared does. It determines how many frames go by _after_ the object has already gone missing before giving up and no longer looking for the object.

If you want stationary object detection, you should look at using the latest beta release.

**Comment by sygys** [comment]
Why does this not work in zones? In zones frigate doesnt seem to wait and does a one on one report when an object is detected or gone. 

Stationary detection sounds very nice! I dont think im going to install beta's though. i guess ill wait until its released in the stable version.

Thanks for all the hard work i really love frigate! If only i could find a coral usb stick... My intel NUC i7 is running pretty hot with this addon :)

**Comment by NickM-27** [comment]
Closing this as a duplicate of https://github.com/blakeblackshear/frigate/issues/15570 which did come later but seems to truly be what is being requested here.

There are too many complications with individual object enabling / disabling via MQTT as well as representing the state of every object that could be enabled or disabled dynamically (IE all 80 objects in the labelmap by default)
